source/tutorials/metadata.rst

```diff
@@ -0,0 +1,141 @@
+Metadata in QIIME 2
+===================
+
+.. note:: This guide assumes you have installed QIIME 2 using one of the procedures in the :doc:`install documents <../install/index>` and completed the :doc:`moving pictures tutorial <moving-pictures>`.
+
+Metadata provides the key to gaining biological insight from your raw data. In QIIME 2, sample metadata includes technical details, such as the DNA barcodes that were used for each sample, or descriptions of the samples, such as which subject, time point and body site each sample came from in a human microbiome time series. Feature metadata is often a feature annotation, such as the taxonomy assigned to a sequence variant. Sample and feature metadata are used by many methods and visualizers in QIIME 2.
+
+.. qiime1-users:: In QIIME 1, metadata (also known as the metadata mapping file), was a user-defined file that contained these study-specific fields. QIIME 2 expands upon this idea, allowing users to provide their own study metadata (via a TSV file), or, by viewing other study-specific artifacts as metadata. Examples of both are presented in the following.
+
+Begin by creating a directory to conduct the work in:
+
+.. command-block::
+   :no-exec:
+
+   mkdir qiime2-metadata-tutorial
+   cd qiime2-metadata-tutorial
+
+Metadata from a text file
+-------------------------
+
+Metadata is typically defined in a sample (or feature; more on that below) metadata mapping file. The QIIME 2 development team hasn't adopted a standard set of criteria for the sample metadata mapping file, but at present the following minimum requirements are enforced:
+
+- The file must be a tab-separated text file
+- Comment lines (e.g. lines that begin with ``#``) are ignored
+- The first non-comment line of the file is used as the column labels
+- The column label values must be unique (i.e. no duplicate values)
+- There must be at least one line of data following the column label line
+- The first column in the table is the "identifier" column (either sample ID or feature ID, depending on the axis)
+- All of the values in the first column must be unique (i.e. no duplicate values)
+
+Sample (and feature) metadata mapping files can be validated using Keemei_, which will help identify issues while creating these files.
+
+.. qiime1-users:: Generally speaking, sample/feature metadata mapping files `that work in QIIME 1`_ should work in QIIME 2.
+
+To get started with understanding sample metadata mapping files, first download an example TSV file:
+
+.. download::
+   :url: https://data.qiime2.org/2017.6/tutorials/moving-pictures/sample_metadata.tsv
+   :saveas: sample-metadata.tsv
+
+This file can be opened and edited in a text editor, Google Sheets, or Keemei_. You can also generate a QIIME 2 visualization of metadata mapping files as follows:
```

Comment:
*Since this is a TSV file, it can be opened and edited in a variety of applications, including text editors, Microsoft Excel, and Google Sheets (e.g. if you plan to validate your metadata with Keemei_).*